appendix-petsciicodes.tex

```diff
@@ -370,104 +376,114 @@ \section{Escape Sequences}
+\specialkey{ESC} \megakey{V} &
 Scrolls the entire screen up one line.\\
 \hline
-\megakey{ESC} \megakey{W} &
+\specialkey{ESC} \megakey{W} &
 Scrolls the entire screen down one line.\\
 \hline
-\megakey{ESC} \megakey{L} &
-Enables scrolling when the cursor down key is pressed at the bottom of the screen.\\
+\specialkey{ESC} \megakey{L} &
+Enables scrolling when \megakey{$\downarrow$} is pressed at the bottom of the screen.\\
 \hline
-\megakey{ESC} \megakey{M} &
-Disables scrolling. When pressing the cursor down key at the bottom on the screen, the cursor will move to the top of the screen. The cursor is restricted at the top of the screen with the Cursor up key.\\
+\specialkey{ESC} \megakey{M} &
+Disables scrolling. When pressing \megakey{$\downarrow$} at the bottom of the screen, the cursor will move to the top of the
+screen. However, when pressing \megakey{$\uparrow$} at the top of the screen, the cursor will remain on the first line.\\
   \hhline{==}
   \multicolumn{2}{l}{\textbf{Insertion and deletion}} \\
   \hhline{==}
-\megakey{ESC} \megakey{I} &
-Inserts an empty line in the current cursor position and moves all subsequent lines down one position.\\
+\specialkey{ESC} \megakey{I} &
+Inserts an empty line at the current cursor position and moves all subsequent lines down one position.\\
 \hline
-\megakey{ESC} \megakey{D} &
-Deletes the current line and moves other lines up one position.\\
+\specialkey{ESC} \megakey{D} &
+Deletes the current line and moves lines below the cursor up one position.\\
 \hline
-\megakey{ESC} \megakey{P} &
-Erases all characters from the cursor to the start of current line.\\
+\specialkey{ESC} \megakey{P} &
+Erases all characters from the cursor to the start of the current line.\\
 \hline
-\megakey{ESC} \megakey{Q} &
-Erases all characters from the cursor to the end of current line.\\
+\specialkey{ESC} \megakey{Q} &
+Erases all characters from the cursor to the end of the current line.\\
   \hhline{==}
   \multicolumn{2}{l}{\textbf{Movement}} \\
   \hhline{==}
-\megakey{ESC} \megakey{J} &
-Moves the cursor to start of current line.\\
+\specialkey{ESC} \megakey{J} &
+Moves the cursor to the start of the current line.\\
 \hline
-\megakey{ESC} \megakey{K} &
-Move to end of the last non-white-space character on the current line.\\
+\specialkey{ESC} \megakey{K} &
+Moves the cursor to the last non-whitespace character on the current line.\\
 \hline
-\megakey{ESC} $\uparrow$ &
-Saves the current cursor position. Use \megakey{ESC} $\leftarrow$ key (next to 1 key) to move back to saved position.\\
+\specialkey{ESC} \megakey{$\uparrow$} &
+Saves the current cursor position. Use \specialkey{ESC} \megakey{$\leftarrow$} to move it back to the saved position.\\
 \hline
-\megakey{ESC} $\leftarrow$ &
-Restores the cursor position to the position stored via a prior press of the \megakey{ESC} $\uparrow$ key (next to RESTORE key).\\
+\specialkey{ESC} \megakey{$\leftarrow$} &

```

Comment:
Similarly, I'd avoid the \megakey in-front of \leftarrow here, just as a way to convey to the reader (we don't mean the left-arrow cursor key, but the left-arrow character key)